### DISCUSSION TITLE
How to make an orbit around the globe?

### DISCUSSION THREAD

**Discussion by GravityMsc** [visgl/deck.gl#5585] [Q&A] [answered]
I want to make an orbit with a satellite(icon) around the globe by using globeView. What should I do? It seems that arcLayer and greatCircleLayer don't work.

**Comment by AdriSolid** [comment, marked as answer]
What did you try?
Here there is an example using `GlobeView` + layers (including `ArcLayer`) with `z`: https://codepen.io/AdriSolid/pen/RwoQeeR
You should be able to configure `z` => `getPosition: d => [d.geometry.coordinates[0], d.geometry.coordinates[1], 5000000]`

**Reply by GravityMsc** [reply]
Actually I want to generate a full circle line around the globe, start position and target position are the same from air by using z, just like a stellite orbit.

**Reply by AdriSolid** [reply]
I did a quick update to the example, adding a `PolygonLayer` as a possible solution for this, check it again, now earth looks like:
![Screenshot from 2021-03-16 13-13-10](https://user-images.githubusercontent.com/25487037/111307388-a6962080-8659-11eb-8707-6b3f0e56f063.png)

**Reply by GravityMsc** [reply]
Ah, that's really clever. I think it is enough good for me. Thank you very much!
